### ISSUE TITLE
Proposed branch deletion: include_hall_dump

### ISSUE THREAD

**Issue by wdconinc** [JeffersonLab/remoll#79]
All the info on the dump geometry that's in that branch is also in develop/geometry. What's in include_hall_dump is probably based on the same (if not older and incorrect) info in prexsim. @cipriangal Is my understanding correct? Is our dump as implemented in dumpDaughter.gdml the most accurate available? If so, can we enable by default? And can it be used (has it been used?) to study backscatter?

**Comment by cameronc137** [comment]
The dumpDaughter.gdml file is the same information as I copied over from Prex II engineering documents and looking carefully at the stuff in @ciprian's prexSim geometry information. It is mostly new implementation, using the slightly different syntax and units (cm instead of mm) of Remoll. The geometry is accurate for Prex but not for MOLLER, but it's at least a starting point (and I have quite a few other versions of the dump geometry sitting in the sbujlab repository). 

I honestly don't remember what I did with the include_dump_geometry branch or if it was even me who did it, but I would assume that the important geometry information for the Prex and Moller dump studies I was doing are stored in my local sbujlab repo instead of the main JLab one. 

This is mostly because I wasn't intending for it to necessarily enter the mainstream geometry configuration, but I included that one file in the main develop/geometry folder so I could at least get rough estimates of radiation impact from the dump (it's not negligible, but not large). 

And yes I suppose it would be useful for splashback studies, but you will need to be very careful about your inclusion and definition of the neckdown pipe and the donut, because they are subject to change quite a bit in the next few years.
